packages/issuer/src/blockchain-facade/CertificateUtils.ts

```diff
@@ -171,13 +172,13 @@ export async function getAllCertificates(
 }
 
 export async function getAllOwnedCertificates(
-    configuration: Configuration.Entity
+    configuration: Configuration.Entity,
+    user: IUserWithRelations
```

Comment:
What's the reason of adding extra `user` paramater, where configuration has already all necessary fields? Seem to me that this PR should just change configuration.activeUser to be set from config instead of MetaMask.